examples/fields/DateField.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { ChronoDateCast, SmartDateField } from './SmartDateField'
```

Comment:
Split this into DateField tests and SmartDateField tests

Reply:
??? I combined the two per your other comment, I think that makes more sense